client/lib/push-notifications/index.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * External dependencies
+ */
+import Emitter from 'lib/mixins/emitter';
+import debugFactory from 'debug';
+import moment from 'moment';
+import store from 'store';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import wpcom from 'lib/wp';
+
+const daysBeforeForcingRegistrationRefresh = 15,
```

Comment:
I'm unsure myself. There are [some](https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/constants.js) [examples](https://github.com/Automattic/wp-calypso/blob/master/client/lib/preferences/constants.js) of that usage, but there's [no guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md) that I can find in that regard.

I'll update it though, because there seem to be more recent examples of this usage than not.